bin/manager_start.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # Script to start the job manager
-# args: <work dir for context> <cluster address> [proxy_user]
+# args: <master> <deployMode> <akkaAdress> <actorName> <workDir> [<proxyUser>]
```

Comment:
Do you think the actor path would be safer and more future-proof than just the actor name?

Reply:
What path did you mean / suggest?

I did not change anything on this:
https://github.com/spark-jobserver/spark-jobserver/blob/306335e76da63190d2b0b2b26f790e7675097bd6/job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala#L222
https://github.com/spark-jobserver/spark-jobserver/blob/master/job-server/src/main/scala/spark/jobserver/JobManager.scala#L75

Reply:
I mean the entire ActorPath that has something like /user/$contextActorName  ....  this matters if we were to put another management actor layer in front of our actors, then the path to look them up might need to be fixed.  If you pass the entire actorPath then it could be used to look up actors remotely even if the hiearchy changes

Reply:
I don't know how to start the actor in the `JobManger` with a given path... Due to the fact that this has nothing to do with cluster mode and providing only the actor name is the same behavior like before, let us address this in another issue/pull request.